_po/ja/how-to/laravel.po

```diff
@@ -324,11 +476,18 @@ msgid ""
 "}\n"
 "```"
 msgstr ""
+"class PostController extends Controller\n"
+"{\n"
+"    //\n"
+"}\n"
+"```"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.